assets/js/components/HealthSummary/HealthSummaryBox.jsx

```diff
@@ -0,0 +1,79 @@
+import {
+  EOS_CHECK_CIRCLE_OUTLINED,
+  EOS_ERROR_OUTLINED,
+  EOS_WARNING_OUTLINED,
+} from 'eos-icons-react';
+import classNames from 'classnames';
+import React from 'react';
+
+const iconByHealth = {
+  passing: () => (
+    <EOS_CHECK_CIRCLE_OUTLINED size={40} className="fill-green-600" />
+  ),
+  warning: () => <EOS_WARNING_OUTLINED size={40} className="fill-yellow-400" />,
+  critical: () => <EOS_ERROR_OUTLINED size={40} className="fill-red-600" />,
+};
```

Comment:
Maybe just a simple switch statement could have worked here?

Reply:
Honestly I think the map approach is more readable, having a switch case returning a function that later I use to render that component in another component 👀 , it's not my cup of tea